lib/utils/user_avatar_color.dart

```diff
@@ -0,0 +1,46 @@
+import "package:flutter/material.dart";
+import "package:proxima/models/ui/linear_segmented_hsv_colormap.dart";
+import "package:proxima/services/database/challenge_repository_service.dart";
+
+/// Stops for the colormap used to color the user's avatar based on their centauri points.
```

Comment:
By humble curiosity, why did you do this refactor?

I'm not a big fan of its current form, because I feel like it adds some namespace pollution. If you don't want to put those lines back to the dynamic user avatar view-model, I think you should at least encapsulate everything inside a class `UserAvatarColor` with a private constructor.

Reply:
please check previous reviews, notably https://github.com/ProximaEPFL/proxima/pull/348#discussion_r1614739378, I think it will answer your question

Reply:
I think it does not make sense to leave it in the dynamic user avatar as it does not handle color anymore.

I think this is linked to this suggestion: https://github.com/ProximaEPFL/proxima/pull/348#discussion_r1614739378

Reply:
Hahahah! I read Yoann's comments and it appears I have managed to miss this one and only comment. That's sad x)

Thank you, I learnt something today :D